packages/Algernon.package/ALGInputField.class/instance/keyboardFocusColor.st

```diff
@@ -0,0 +1,3 @@
+accessing
+keyboardFocusColor
+	^ ALGBreadcrumbs defaultColor
```

Comment:
Is that the correct color to use ?

Reply:
This behaivior was adapted from ALGCore. That's there the focus lay before these changes. But feel free to suggest another color.
(This color is chosen not to interfere with the breadcrumbs.)

Reply:
Okay, all right, I was just wondering what this color has to do with the focus.

Reply:
I think we should add a comment explaining that intent. Could you do that @robertkirsten? (I do not want to create merge conflicts.) I'll write the comment for you
> Make the keyboard focus color match the background it might cover to hide it.

Reply:
Done.